fabfile.py

```diff
@@ -191,8 +191,9 @@ def _build(ts, name, registry):
             local("sudo docker push %s/%s:%s" % (registry, name, ts))
         local("sudo docker rmi %s/%s:%s" % (registry, name, ts))
 
-def _set_image(ts, name, job):
-    local("kubectl set image -n metallb-system {2} {1}={3}/{1}:{0}".format(ts, name, job, _registry_clusterip()))
+def _set_image(ts, name, job, registry):
+    set_from_registry = _registry_clusterip() if "localhost" in registry else registry
```

Comment:
set_from_registry is unused. Remove?

Reply:
I wasn't using it but I should :) Now it's fixed.